src/reps/prop-rep.js

```diff
@@ -48,8 +53,11 @@ let PropRep = React.createClass({
     let key;
     // The key can be a simple string, for plain objects,
     // or another object for maps and weakmaps.
-    if (typeof this.props.name === "string") {
-      key = span({"className": "nodeName"}, this.props.name);
+    if (typeof name === "string") {
+      if (!this.props.suppressQuotes) {
```

Comment:
nit: get suppressQuotes from props in the destructuring assignment above and use if(!suppressQuotes) here